extensions.json

```
@@ -1506,10 +1506,8 @@
     },
     {
       "id": "rebornix.ruby",
-      "repository": "https://github.com/rubyide/vscode-ruby",
-      "version": "0.28.1",
-      "checkout": "v0.28.1",
-      "location": "packages/vscode-ruby-client"
+      "download": "https://github.com/rubyide/vscode-ruby/releases/download/v0.28.0/ruby-0.28.0.vsix",
+      "version": "0.28.0"
```

Sad, they forgot to upload the `ruby-0.28.1.vsix`. Undoing this:

```suggestion
      "download": "https://github.com/rubyide/vscode-ruby/releases/download/v0.28.0/ruby-0.28.0.vsix",
      "version": "0.28.1",
      "checkout": "v0.28.1",
      "location": "packages/vscode-ruby-client"
```